modules/document_repository/README.md

```diff
@@ -20,7 +20,8 @@
 
 ## Configurations
 
-- The Document Repository enables users to upload and organize project files of any type that can easily be viewable for users with appropriate permissions.Give full permissions to store documents on server. To give full permissions to store documents on the server: 
+- The Document Repository enables users to upload and organize project files of any type that can easily be viewable for users with appropriate permissions.
```

Comment:
Could we elaborate on what "of any type" means?

Also .. "that can easily be viewed by users with the appropriate permissions."